recipe/bld.bat

```diff
@@ -0,0 +1,36 @@
+set CMAKE_CONFIG=Release
+
+mkdir build_shared_%CMAKE_CONFIG%
+pushd build_shared_%CMAKE_CONFIG%
+
+cmake -G "NMake Makefiles" ^
+      -DCMAKE_BUILD_TYPE:STRING=%CMAKE_CONFIG% ^
+      -DBUILD_SHARED_LIBS:BOOL=ON ^
+      "%SRC_DIR%"
+nmake
+ctest -V
+
+move bro.exe "%LIBRARY_BIN%\"
+move *.dll "%LIBRARY_BIN%\"
```

Comment:
This line confuses me a bit, but this seems to be difference on Windows that DLLs and Executables live in the same folder?